webanno-diag/src/main/java/de/tudarmstadt/ukp/clarin/webanno/diag/repairs/ReattachFeatureAttachedSpanAnnotationsAndDeleteExtrasRepair.java

```diff
@@ -102,7 +102,7 @@ else if (candidate != anno) {
             
             // Delete those the extra candidates that are not properly attached
             if (!toDelete.isEmpty()) {
-                toDelete.forEach(fs -> aCas.removeFsFromIndexes(fs));
```

Comment:
I was a bit surprised though that apparently only this single problematic instance was changed because this `t -> someMethod(t)` happens a lot in AnnotationPage, CurationPage, etc.